src/App.js

```diff
@@ -1,14 +1,37 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import Overlay from "./Components/Overlay";
 import SearchBar from "./Components/SearchBar";
-
+import axios from "axios";
 import "./App.css";
+
 function App() {
-  const [search, setSearch] = useState(false);
+  const [searchTerm, setSearchTerm] = useState("");
+  const [searchResult, setSearchResults] = useState([]);
+  const dataFetch = async () => {
```

Comment:
Rename this function. Make sure the name implies what you're achieving with the function. Eg: `fetchWordSearchResults`.